custom_components/smart_envi/services.py

```
@@ -0,0 +1,386 @@
+                "current_temperature": device_info.get("ambient_temperature"),
+                "target_temperature": device_info.get("current_temperature"),
+                "state": "on" if device_info.get("state") == 1 else "off",
+                "mode": device_info.get("current_mode"),
+                "firmware_version": device_info.get("firmware_version"),
+                "signal_strength": device_info.get("signal_strength"),
+            }
+        except Exception as e:
+            _LOGGER.error("Failed to get status for %s: %s", entity_id, e, exc_info=True)
+            return None
+
+    async def test_connection(call: ServiceCall) -> None:
```

Ruff (0.14.8)

Unused function argument: `call`

(ARG001)